module/enrichers.mjs

```diff
@@ -718,37 +775,51 @@ export function createRollLabel(config) {
 /* -------------------------------------------- */
 
 /**
- * Create a rollable link.
- * @param {string} label    Label to display.
- * @param {object} dataset  Data that will be added to the link for the rolling method.
+ * Create a rollable link with a request section for GMs.
+ * @param {HTMLElement|string} label  Label to display
+ * @param {object} dataset            Data that will be added to the link for the rolling method.
  * @returns {HTMLElement}
  */
-function createRollLink(label, dataset) {
+function createRequestLink(label, dataset) {
   const span = document.createElement("span");
-  span.classList.add("roll-link");
+  span.classList.add("roll-link-group");
   _addDataset(span, dataset);
-
-  // Add main link
-  const link = document.createElement("a");
-  link.dataset.action = "roll";
-  link.innerHTML = '<i class="fa-solid fa-dice-d20"></i>';
-  link.append(label);
-  span.insertAdjacentElement("afterbegin", link);
+  if (label instanceof HTMLElement) span.insertAdjacentElement("afterbegin", label);
+  else span.insertAdjacentHTML("afterbegin", label);
```

Comment:
Need to be careful to not parse arbitrary user-entered strings as HTML. Using `append` here to append a string ensures that it is rendered as a text node, and not HTML.

```suggestion
  if ( label instanceof HTMLElement ) span.insertAdjacentElement("afterbegin", label);
  else span.append(label);
```